command/agent/config.go

```diff
@@ -548,6 +552,53 @@ type RaftBoltConfig struct {
 	NoFreelistSync bool `hcl:"no_freelist_sync"`
 }
 
+// PlanRejectionTracker is used in servers to configure the plan rejection
+// tracker.
+type PlanRejectionTracker struct {
+	// Enabled controls if the plan rejection tracker is active or not.
+	Enabled bool `hcl:"enabled"`
```

Comment:
If we ever want to default this to `true` we'll have to change the type to a pointer (`*bool`) to differentiate unset vs explicitly-false. That shouldn't be a problem though since unlike objects sent over RPCs we never have to worry about an agent on the new version sending a `nil` to an agent still expecting a true or false.

Reply:
Good point, I may as well do the work now 😄 
https://github.com/hashicorp/nomad/pull/13421/commits/f5936f0830e1b5a490a8a4f5f7d39f138cd628a3